src/Altinn.Platform/Altinn.Platform.Storage/Storage.Interface/Models/InstanceOwnerLookup.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Storage.Interface.Models;
+
+namespace Altinn.Platform.Storage.Models
+{
+    /// <summary>
+    /// Model to hold a instance owner lookup element
+    /// </summary>
+    [Serializable]
+    public class InstanceOwnerLookup
+    {
+        /// <summary>
+        /// Person number (national identification number)
+        /// </summary>
+        [JsonProperty(PropertyName = "personNumber")]
+        public string PersonNumber { get; set; }
+
+        /// <summary>
+        /// organization number
+        /// </summary>
+        [JsonProperty(PropertyName = "organizationNumber")]
+        public string ElementType { get; set; }
```

Comment:
organization => organisation

Reply:
Also, should ElementType be OrganisationNumber, or is it correct as is?

Reply:
yup. I fix.